repair/row_level.cc

```diff
@@ -1379,7 +1379,7 @@ class repair_meta {
         if (rows.empty()) {
             return;
         }
-        auto row_diff = to_repair_rows_list(rows).get0();
+        auto row_diff = to_repair_rows_list(std::move(rows)).get0();
```

Comment:
LGTM
suggestion: we can change the interface to accept a rvalue reference to make this clear...
same with `to_repair_rows_list`

Reply:
I started to get the feeling that with coroutines, rvalue reference parameters fell out of vogue, and we started to use by-value parameters, including moving into them.
Anyway, I'll commit this fix as-is and let Asias consider the interface change for a followup patch.

Reply:
I personally like the plain func(Foo foo) signature style. It is up to caller to decide if it wants to copy or move the object.